indico/modules/events/timetable/templates/pdf/_program.html

```diff
@@ -0,0 +1,340 @@
+                {% if type == 'BREAK' %}
+                    {% trans %}Break{% endtrans %}
+                {% elif type == 'CONTRIBUTION' %}
+                    {% trans %}Contribution{% endtrans %}
+                {% elif sess.is_poster %}
+                    {% trans %}Poster Session{% endtrans %}
+                {% else %}
+                    {% trans %}Session{% endtrans %}
+                {% endif %}
+            </span>
+        </p>
+        <p>
+            <span class="bold">{% trans %}Location{% endtrans %}:</span>
+            {{ render_place(entry) }}
+        </p>
+        {% set people = object.person_links if type == 'SESSION_BLOCK' else object.speakers %}
+        {% if type != 'BREAK' and people %}
+            <span class="bold">
+                {% if type == 'SESSION_BLOCK' %}
+                    {{ ngettext('Convener:', 'Conveners:', people|length) }}
+                {% else %}
+                    {{ ngettext('Speaker:', 'Speakers:', people|length) }}
+                {% endif %}
+                &nbsp;
+            </span>
+            {{ render_names(people, show_title=config.show_title, show_affiliation=config.show_affiliation) }}
+        {% endif %}
+    </div>
+{% endmacro %}
+
+{% macro render_block_date(entry, day, print_date_close_to_sessions) %}
+    {% if print_date_close_to_sessions %}
+        <p class="session-block-date">
+            <span class="bold">{{ day|format_skeleton('EEEEdMMMM', timezone=entry.event.timezone) }}</span>
+            <span class="timebox">
+                {{ format_interval(entry.start_dt.astimezone(entry.event.tzinfo), entry.end_dt.astimezone(entry.event.tzinfo), format='HHmm') }}
+            </span>
+        </p>
+    {% endif %}
+{% endmacro %}
+
+{% macro render_time(entry) %}
+    <div class="time">
+        <div class="time-from">
+            {{ entry.start_dt|format_time(timezone=entry.event.timezone) }}
+        </div>
+        <div class="time-divider">
+            {# Line between generated in CSS #}
+        </div>
+        <div class="time-divider">
+            {# Line between generated in CSS #}
+        </div>
+        <div class="time-to">
+            {{ entry.end_dt|format_time(timezone=entry.event.timezone) }}
+        </div>
+    </div>
+{% endmacro %}
+
+{# Utilities #}
+
+{% macro render_place(entry) %}
+    {% set entry = entry.object if entry.object else entry %}
+
+    {{ [entry.venue_name, entry.room_name, entry.address] | select | join(', ') }}
+{% endmacro %}
+
+{% macro render_names(people, show_title=false, show_affiliation=false) %}
+    {% for person in people %}
+        {%- if show_title %}
+            {{ person.title }}
+        {%- endif %}
+        {{ person.display_full_name }}
+        {%- if show_affiliation and person.affiliation %}
+            ({{ person.affiliation }})
+        {%- endif -%}
+        {%- if not loop.last %},{% endif %}
+    {% endfor %}
+{% endmacro %}
+
+{% macro get_custom_id(entry) -%}
```

Comment:
I recommend prefixing macros which are only used inside this file with an `_`.  In Jinja, such macros are not exported so when making changes to them in the future you'll immediately know that you won't have to search in any other files for usages of them.

Reply:
Ah interesting, didn't know. Okay, thanks!